bitnami/kube-prometheus/Chart.yaml

```diff
@@ -34,4 +34,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-prometheus
   - https://github.com/bitnami/bitnami-docker-alertmanager
   - https://github.com/prometheus-operator/kube-prometheus
-version: 4.0.1
+version: 4.0.2
```

Comment:
as it is a feature, could you increase the version to 4.1.0?

Reply:
Done! 😄